release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1029,6 +1029,7 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 
 * Previously, systemd was unable to read environment files in `/etc/kubernetes`. The SELinux policy caused this and as a result, the kubelet did not start. The policy has been modified. The kubelet starts and the environment files are read. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969998[*BZ#1969998*])
 
+* In an s390x Kernel Virtual Machine (KVM) with an ECKD DASD attached, the DASD would appear to be a regular virtio storage device but would become inaccessible if the VTOC was removed. As a result, you could not use DASD as a virtio block device when installing {op-system-first} on the KVM. The `coreos-installer` program has been updated so that it now installs RHCOS with a VTOC-format partition table when the installation destination is a virtio storage device such as an ECKD DASD attached to a KVM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960485[*BZ#1960485*])
```

Comment:
Are these initialisms known well enough not to write out? ECKD, DASD, VTOC? I don't know the space well enough myself: If you're happy, I'm happy.

Otherwise, LGTM.

Reply:
So I went through and added Kernel Virtual Machine (KVM) because it was an initialism. For DASD and ECKD, I searched the repo for them and only the initialisms came up. My assumption then was that they're OK. I'll ask Bob.

Reply:
I agree with @stevsmit and the change here. 

(In the few instances of DASD ECKD in our docs, we don't spell out, probably because they are specific to IBM Z hardware and more familiar to those users. 🤷  For RNs, I see less formality about spelling out some of these initialisms due to the high volume of bug fix reports.)